### ISSUE TITLE
Album artist not available on playlists?

### ISSUE THREAD

**Issue by drdownload** [jrnewell/spotify-ripper#190]
Hi,

one question, could it be possible that the album artist information is not availabe if I collect some albums in a playlist and run the playlist through spotify-ripper?

is there a way to realize this with the formating / metadata string options?

**Comment by drdownload** [comment]
Hmm, If I see that correctly if the album is none the code skips any further check for album artist, if I remember the api correctly we would need to loop back to the album object of the track and get the original album artist.

About a year ago I had an accident and patched spotify ripper with some custom compilation support (album type=2 or album artist "various artists" ;) to use an alternative naming pattern and set the proper compilation id3 flag for itunes.

any interest that I create a pull request with alternative compilation handling and maybe (if i can get it to work) support for album artists on playlists?

**Comment by jrnewell** [comment]
yeah sure :smile: I am not sure 100% I followed what you did, but I be more than happy to accept any contributions you have.

**Comment by bhelm** [comment]
i would like the "Various Artist" option to make a playlist into a "custom" album with correct naming and ID3.
